rpm/htcondor-ce.spec

```diff
@@ -190,21 +190,6 @@ rm $RPM_BUILD_ROOT%{_unitdir}/condor-ce{,-collector}.service
 rm $RPM_BUILD_ROOT%{_tmpfilesdir}/condor-ce{,-collector}.conf
 %endif
 
-# Directories necessary for HTCondor-CE files
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/run/condor-ce
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/log/condor-ce
-install -m 1777 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/log/condor-ce/user
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lib/condor-ce
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lib/condor-ce/spool
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lib/condor-ce/spool/ceview
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lib/condor-ce/spool/ceview/vos
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lib/condor-ce/spool/ceview/metrics
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lib/condor-ce/execute
-install -m 0755 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lock/condor-ce
-install -m 1777 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lock/condor-ce/user
-install -m 1777 -d -p $RPM_BUILD_ROOT/%{_localstatedir}/lib/gratia/condorce_data
```

Comment:
Does the build still succeed without all these `install`s in the `%install` section?  (I get that the idea is to do the dir creation in cmake instead, just wondering if it works as expected.)

Reply:
We build the RPM in the CI, so yes: https://travis-ci.org/opensciencegrid/htcondor-ce/builds/423088342?utm_source=github_status&utm_medium=notification

Reply:
Sounds good then.